lib/3scale_toolbox/commands/import_command/openapi/method.rb

```diff
@@ -12,12 +12,16 @@ def method
           end
 
           def friendly_name
-            operation[:operationId]
+            operation[:operationId] || operation_id
           end
 
           def system_name
             friendly_name.downcase
           end
+
+          def operation_id
+            "#{operation[:verb]}#{operation[:path].tr('/{}', '')}"
```

Comment:
Woudn't it be safer to strip out everything other than `/\w/` ? `.gsub(/[^\w]/, '')`

Reply:
done